ta/crypto_perf/ta_crypto_perf.c

```diff
@@ -29,6 +29,10 @@ static int use_iv;
 
 static TEE_OperationHandle crypto_op;
 static uint32_t algo;
+static TEE_OperationHandle crypto_op_enc_sign;
+static TEE_ObjectHandle crypto_obj;
+static TEE_Attribute *attrs;
+static uint32_t attr_count;
```

Comment:
could you rename `attrs` and `attr_count`. They match local variable names which is confusing.
Maybe `asym_perf_attrs`/`asym_perf_attr_count`?